src/main.ts

```diff
@@ -257,9 +256,9 @@ autoUpdater.on(
 )
 
 autoUpdater.on('update-not-available', () => {
-  console.log('update-not-available')
+  console.warn('update-not-available')
 })
 
 autoUpdater.on('error', err => {
-  console.log('error', { err })
+  console.warn('autoUpdater error: ${err.message}', err)
```

Comment:
Could you please log this error everywhere in this PR? I see that like [#258](https://github.com/pastelnetwork/pastel-electron-wallet/pull/61/files/303efb93e167796023dc2d0c03f3aa199935a544..de08638d17d097333064a65af5824c089bfce818#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR258) is still wrong

Reply:
OK @br4in3x , I have removed the un-need console.log